src/WebJobs.Extensions.DurableTask/Scale/DurableTaskTriggersScaleProvider.cs

```diff
@@ -0,0 +1,127 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+#nullable enable
+#if FUNCTIONS_V3_OR_GREATER
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using Microsoft.Azure.WebJobs.Host.Scale;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask.Scale
+{
+    internal class DurableTaskTriggersScaleProvider : IScaleMonitorProvider, ITargetScalerProvider
+    {
+        private IScaleMonitor monitor;
+        private ITargetScaler targetScaler;
+
+        public DurableTaskTriggersScaleProvider(
+            IServiceProvider serviceProvider,
+            TriggerMetadata triggerMetadata)
+        {
+            string functionId = triggerMetadata.FunctionName;
+            FunctionName functionName = new FunctionName(functionId);
+
+            var options = this.GetOptions(serviceProvider, triggerMetadata);
+
+            var durabilityProviderFactory = this.GetDurabilityProviderFactory(serviceProvider, options);
+            var defaultDurabilityProvider = durabilityProviderFactory.GetDurabilityProvider();
```

Comment:
Please use explicit types instead of `var` when it's not obvious what the type is.

Reply:
Thanks.  Yeah I see I've been doing this. Will be addressed in my next commit